frontend/src/features/projects/spl/forms/CloseOutFinancialSummaryForm.tsx

```diff
@@ -124,21 +123,21 @@ const CloseOutFinancialSummaryForm = (props: CloseOutFinancialSummaryFormProps)
         <Col md={1}>&nbsp;</Col>
         <Col>
           <ProjectNotes
-            field={`notes[${NoteTypes.Remediation}].note`}
+            field="remediationNote"
             label="Remediation (Not funded throuch COS. TRAN only)"
             disabled={props.isReadOnly}
             outerClassName="col"
             className="col-md-10"
           />
           <ProjectNotes
-            field={`notes[${NoteTypes.SplCost}].note`}
+            field="programCostNote"
             label="SPL Cost Calculation Notes"
             disabled={props.isReadOnly}
             outerClassName="col"
             className="col-md-10"
           />
           <ProjectNotes
-            field={`notes[${NoteTypes.SplGain}].note`}
```

Comment:
Using array positioning is an invalid implementation and will result in data corruption